cms/models.py

```diff
@@ -707,10 +720,13 @@ class HomeProductLink(models.Model):
         blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
+        verbose_name="Featured Product Page",
```

Comment:
Again - this was the only change to the model.

Reply:
Wagtail does this for us in the CMS :) "Only the following page types may be chosen for this field: Course page, Program page. Search results will exclude pages of other types."

Reply:
Yes, there is a message in CMS. I mainly mentioned it here for developer who look at model because field name is course_product_page so my instinct would be its only for course. Not big deal, just for clarification.

Reply:
I can add in a comment about hte historical context/name. Help text shows to user and that page is already busy as heck.

Reply:
Added.